app/models/name_sorter.rb

```diff
@@ -129,14 +129,15 @@ def only_single_names
   end
 
   def push_synonym(arg)
-    if arg.is_a?(Integer)
+    case arg
+    when Integer
       @approved_synonyms.push(Name.find(arg))
-    elsif arg.is_a?(ActiveRecord::Base)
+    when ActiveRecord::Base
       @approved_synonyms.push(arg)
     else
       raise(TypeError.new(
         "NameSorter synonyms must be Integer or ActiveRecord::Base, "\
-        "not #{arg.clasS}."
+        "not #{arg.class}."
```

Comment:
Fixes unrelated bug I spotted. 
Part of the reason this bug is here is that this line is uncovered.
So I plan to: revert this fix, add a new PR which makes the fix and covers this line.

Reply:
Story created: https://www.pivotaltracker.com/story/show/174349504